performance-tests/remotewb-performance-tests/addUsers.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+$JBOSS_HOME/bin/add-user.sh -a perfUser 'perfUser1234;'
+$JBOSS_HOME/bin/add-user.sh -a engUser 'engUser1234;'
+echo "perfUser=admin" >> $JBOSS_HOME/standalone/configuration/application-roles.properties
+echo "engUser=admin,engineering" >> $JBOSS_HOME/standalone/configuration/application-roles.properties
```

Comment:
BTW, the reason behind my thoughts is that it's always nice if people can simply run the whole test using "mvn install"... The easier it is to run, the more other developers will use and support the test.